src/System.Text.Json/tests/WritableJsonApiTests.cs

```diff
@@ -255,16 +368,94 @@ public static void TestAddingKeyValuePairsCollectionAfterInitialization()
         {
             var employees = new JsonObject();
             employees.AddRange(EmployeesDatabase.GetTenBestEmployees());
+
+            string prevId = "";
+            foreach ((string id, JsonNode employee) in employees)
+            {
+                Assert.NotEqual(prevId, id);
+                prevId = id;
+
+                Assert.IsType<JsonObject>(employee);
+            }
         }
 
         /// <summary>
-        /// Creating Json array
+        /// Adding JsonArray to JsonObject by creating it in initializing collection
         /// </summary>
         [Fact]
-        public static void TestCreatingJsonArray()
+        public static void TestAddingJsonArray()
         {
-            string[] dishes = { "sushi", "pasta", "cucumber soup" };
+            var preferences = new JsonObject()
+                {
```

Comment:
NIT: extra spaces before the bracket, and also you have spaces after the line `.. "red", "green", "purple" } }      `